main/init/test/src/mill/init/maven/InitModuleTests.scala

```diff
@@ -0,0 +1,134 @@
+package mill.init.maven
+
+import mill.testkit.{TestBaseModule, UnitTester}
+import utest.*
+
+object InitModuleTests extends TestSuite {
+
+  def tests: Tests = Tests {
+
+    val sourceRoot: os.Path = os.Path(sys.env("MILL_TEST_RESOURCE_DIR")) / "maven"
+
+    object initmodule extends TestBaseModule with InitModule
+
+    test("single") {
+
+      val eval = UnitTester(initmodule, sourceRoot / "single-module")
+
+      eval(initmodule.init()).isRight ==> true
+
+      os.walk(initmodule.millSourcePath).count(_.ext == "mill") ==> 1
+      os.exists(initmodule.millSourcePath / "build.mill") ==> true
+
+      val buildContents = os.read(initmodule.millSourcePath / "build.mill")
+
+      buildContents.contains(
+        s"""  override def pomSettings = PomSettings(
+           |    s\"\"\"Sample single module Maven project with a working, deployable site.\"\"\".stripMargin,
+           |    "",
+           |    "http://www.example.com",
+           |    Seq(),
+           |    VersionControl(
+           |      Option("http://github.com/gabrielf/maven-samples"),
+           |      Option("scm:git:git@github.com:gabrielf/maven-samples.git"),
+           |      Option("scm:git:git@github.com:gabrielf/maven-samples.git"),
+           |      Option("HEAD")
+           |    ),
+           |    Seq()
```

Comment:
Let's generate parameter names here, since that'll make it much clearer what things like `""` and `Seq(0` represent or what the difference is between the two `Option("scm:git:git@github.com:gabrielf/maven-samples.git")`

Also let's use `Some()` instead of `Option()`